decidim-collaborative_texts/spec/commands/decidim/collaborative_texts/admin/publish_document_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module CollaborativeTexts
+    module Admin
+      describe PublishDocument, type: :command do
+        let(:organization) { create(:organization, available_locales: [:en]) }
+        let(:document) { create(:collaborative_text_document, published_at: nil, title: "This is and original document test title") }
+        let(:current_user) { create(:user, :admin, :confirmed, organization:) }
+        let(:command) { described_class.new(document, current_user) }
+
+        describe "call" do
+          context "when the collaborative text document is not already published" do
+            it "publishes the collaborative text document" do
+              expect(Decidim.traceability).to receive(:perform_action!).with(
+                :publish,
+                document,
+                current_user,
+                visibility: "all"
+              ).and_call_original
+
+              expect(command).to broadcast(:ok, document)
```

Comment:
you have no test checking that the published_at actually changes 
```suggestion
              expect(command).to broadcast(:ok, document)
              expect(document.reload).to be_published
```

Reply:
Done